pennylane/interfaces/__init__.py

```diff
@@ -56,8 +54,12 @@
 2. It correctly passes the quantum analytic gradient to the classical machine
    learning library during backpropagation.
 
-We now support additional classical interfaces in PennyLane,
-beginning with PyTorch and TensorFlow eager execution mode.
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    Creating QNodes <self>
```

Comment:
This one looks a bit off in the rendered docs.

I see
- The QNode decorator [links to the page titled "The QNode decorator"]
  - Creating QNodes [links to the page titled "The QNode decorator"]
  - NumPy interface [the page titled "NumPy interface"]
  - PyTorch interface [the page titled "PyTorch interface"]
  - TensorFlow eager interface [the page titled "TensorFlow eager interface"]

Meanwhile, there is a section on the "The QNode decorator" page titled "Classical interfaces"

The main issue is that there is this fictitious heading in the TOC called "Creating QNodes" (which points to a duplicate of its header page), whereas the actual heading "Classical interfaces" does not appear in the TOC at all

Reply:
Yeah, there is definitely something wrong here.

Reply:
Let me know how that looks... tried to fix it up, but the way Sphinx autopopulates the sidebar TOC is odd.

Reply:
![image](https://user-images.githubusercontent.com/2959003/63170500-f969b980-c06b-11e9-8c55-3eb1ce850a64.png)

Reply:
![image](https://user-images.githubusercontent.com/2959003/63212275-73657580-c134-11e9-8d8d-e1b4696f9aba.png)